project/webapp/templates/thread.html

```diff
@@ -90,4 +90,12 @@
     //
     // if (cw.ws.readyState == WebSocket.OPEN) cw.ws.onopen();
 </script>
+
+<script>
+  $(document).ready(function() {
+    // Add current thread ID to export button href
+    document.getElementById('csv-export').href = '/civi2csv?thread=' + /[^/]*$/.exec(window.location.href)[0];
```

Comment:
This is not necessary either. You can pass in the `threadId` as a URL parameter, defined in the route. See comment above.